osf/utils/migrations.py

```diff
@@ -390,6 +390,39 @@ def create_schema_blocks_for_question(state, rs, question, sub=False):
         split_options_into_blocks(state, rs, question, schema_block_group_key)
 
 
+
+def create_schema_blocks_for_simple_schema(schema):
+    from osf.models import RegistrationSchemaBlock
+    schema_block_group_key = ''
+    page_num = 0
+    q_num = 1
+    for block in schema.schema['blocks']:
+        block_type = block['block_type']
+
+        if block_type == 'question-label':
+            schema_block_group_key = generate_object_id()
+        if block_type == 'section-heading':
+            schema_block_group_key = generate_object_id()
+        if block_type == 'subsection-heading':
+            schema_block_group_key = generate_object_id()
```

Comment:
Looking at https://api.osf.io/v2/schemas/registrations/5730e99a9ad5a102c5745a8a/schema_blocks/ as an example of an existing schema with section-headings, it seems that only question-labels (and their corresponding input fields) get the schema_block_group_key.